compiler/qsc_project/src/manifest.rs

```diff
@@ -0,0 +1,81 @@
+use crate::Error;
+use serde::Deserialize;
+use std::path::PathBuf;
+use std::{
+    env::current_dir,
+    fs::{self, DirEntry, FileType},
+};
+
+pub const MANIFEST_FILE_NAME: &str = "qsharp.json";
+
+#[derive(Deserialize, Debug, Default)]
+pub struct Manifest {
+    pub author: Option<String>,
+    pub license: Option<String>,
+    #[serde(default)]
+    pub exclude_files: Vec<String>,
```

Comment:
We do want the ability to include files from anywhere on the filesystem, right? (At least, that's what I recall from the group discussion - hope I didn't get mixed up). In that case we probably need an include pattern as well.